draft-ietf-acme-acme.md

```diff
@@ -2876,11 +2895,15 @@ Label column of the ACME Identifier Types registry.
 Template:
 
 * Label: The identifier for this validation method
-* Identifier Type: The type of identifier that this method applies to
+* Identifier Type: The type of identifier that this method applies to.
```

Comment:
This template section is inconsistent with respect to the trailing `.` - "Identifier Type" and "ACME" use a trailing period, the "Label" and "Reference" bullets do not. I don't have a preference except that they be consistent :-)